options/rtdl/generic/linker.cpp

```diff
@@ -462,9 +560,11 @@ void ObjectRepository::_parseDynamic(SharedObject *object) {
 		}
 	}
 
-	if(runpath_offset)
+	if(runpath_offset) {
 		object->runPath = reinterpret_cast<const char *>(object->baseAddress
 				+ object->stringTableOffset + *runpath_offset);
+		// TODO process tokens and add to global library list
```

Comment:
?

Reply:
outdated, misunderstanding in the semantics of rpath